### PR TITLE
Renovated and Validated bill creation

### LINKED ISSUE #97: Feature Request: Form validation
**User Story**
As a user,
I want to have my forms validated,
So I know when I did not fill them out correctly

**Priority**
[]: High
[x]: Medium
[]: Low

**Describe the solution you want to implement**

- The user is informed that the fields cannot be empty if any of the submitted fields are empty
- The user is informed if their login/password pair is invalid
- During sign up, the user is informed if their login credentials already exist in the database

Please add to this list if there is any other scenario where a form needs to be validated

**Describe alternatives you've considered**
n/a

**Issue Dependencies**
n/a 

**Additional context**
n/a

### LINKED ISSUE #99: Bug Report: Adding the same name multiple times
**Describe the bug**
When adding a bill, the user can add multiple participants with the same name. This should not be allowed.

**To Reproduce**
1. Log in
2. Go to 'Split a bill'
3. Attempt to add multiple participants with the same name
4. To see potential issues, select one of the participants of the same name as a payee
5. As a result, all participants with this name will be selected as a payee 

**Severity**
[]: Minor effect eg. graphical
[x]: Functional error eg. App does not function correctly
[]: Severe eg. Crashing

**Reproducibility**
[x]: Consistent
[]: Occasional
[]: Cannot reproduce at the moment eg. unsure

**Expected behavior**
The user is notified that there is a participant with this name already 

**Screenshots, Images and Traces**
If applicable, add screenshots, images or stack Traces to help explain your problem.
image

**Conditions**
Not dependent on any external conditions (to my knowledge)

**Issue Dependencies**
n/a

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by vee-christella**
Fixes informal usability requests and error handling (temporary way for form validation).

Closes #99 
Closes #97 

- [x] **NOTE: @williamlixu must push in his logical changes for this to rebase off before this can be merged!** 

**Description**

- Styled and renovated the Create Bills page
- Added restrictions to input types for bill amounts
- Added error reports if user inputs an invalid/duplicate name
- Added more error reports if payee isn't selected or if not enough people are added to split the bill
- For some reason the required text inputs didn't throw an error if empty so I added a manual error alert

**Testing**

ESLint tests and visual tests.

**Checklist**

- [x] I have tested my change
- [x] Code builds successfully and all tests pass
- [ ] Docs have been added/updated if needed

### CODE REVIEW

File: split-webapp/split/src/pages/Split.js [hunk 1]
```diff
@@ -32,6 +32,17 @@ class Split extends Component {
     const name = document.getElementById("name").value;
     document.getElementById("name").value = "";
 
+    if (name.length === 0) {
+      alert("Please enter a valid non-empty name.");
```
Comment (williamlixu) [inline split-webapp/split/src/pages/Split.js:36]: Using alerts is probably not the cleanest way to do the validation but I think it's fine, the next group can make it look nicer if they want :)

**Review by williamlixu — APPROVED** [review]
Yoo looks awesome! Good work.

**Review by azizul-i — APPROVED** [review]
Thanks for the great additions in terms of UI and error handling!! It looks great! (The new split button and dollar symbol near cost is awesome )

**Review by jake-good — APPROVED** [review]
Very good :) Only comment is maybe looking to have some form of inbuilt validation that does not cause alerts in the future, but should be fine for now as is 👍

**Comment by williamlixu** [comment]
Just fixed the merged conflict in the `.css` file to merge it.

edit: Actually screwed up then @vee-christella :1st_place_medal: fixed it for real

**Pull request merged**